Source/Core/Core/PowerPC/Jit64Common/Jit64AsmCommon.cpp

```diff
@@ -340,6 +340,9 @@ alignas(16) static const float m_m128 = -128.0f;
 // Sizes of the various quantized store types
 constexpr std::array<u8, 8> sizes{{32, 0, 0, 0, 8, 16, 8, 16}};
 
+// TODO: Use the actual instruction being emulated (needed for alignment exception emulation)
+static const UGeckoInstruction ps_placeholder_instruction = 0;
```

Comment:
Could someone who knows Jit64 better give me some advice here? Basically, I want to pass an additional argument to the quantized loadstore routines (so that the slowmem handler then can forward the argument when calling `Read_*` or `Write_*`), but all of `RSCRATCH`, `RSCRATCH2` and `RSCRATCH_EXTRA` are already used for arguments to the quantized loadstore routines. How should I pass the argument?